tests/dev/script.R

```diff
@@ -12,15 +9,28 @@ vec <- c(parameter, parameter)
 addOutputs("Organism|*|Plasma|Caffeine", simulation = sim)
 addOutputs(parameter, simulation = sim)
 
-print(outputSelections)
+#print(outputSelections)
 
 # parameter <- getParameter("Organism|Liver|Volume", sim)
 # print(parameter)
 
-# results <- runSimulation(sim, population)
+results <- runSimulation(sim)
+exportResultsToCSV(results, sim, "C:/temp/export/results.csv")
 
+pkAnalyses <- getPKAnalyses(results, sim)
 
-# getPkAnalysis(results, sim)
+exportPKAnalysesToCSV(pkAnalyses, sim, "C:/temp/export/pk.csv")
+
+pkParameters<- pkAnalyses$allPKParametersFor("Organism|PeripheralVenousBlood|Caffeine|Plasma (Peripheral Venous Blood)")
+
+for (pkParameter in pkParameters) {
+  print(pkParameter)

```

Comment:
Output in console

 Organism|PeripheralVenousBlood|Caffeine|Plasma (Peripheral Venous Blood): C_max
  Organism|PeripheralVenousBlood|Caffeine|Plasma (Peripheral Venous Blood): C_max_norm
  Organism|PeripheralVenousBlood|Caffeine|Plasma (Peripheral Venous Blood): t_max
  Organism|PeripheralVenousBlood|Caffeine|Plasma (Peripheral Venous Blood): C_tEnd
  Organism|PeripheralVenousBlood|Caffeine|Plasma (Peripheral Venous Blood): AUC
  Organism|PeripheralVenousBlood|Caffeine|Plasma (Peripheral Venous Blood): AUC_norm
  Organism|PeripheralVenousBlood|Caffeine|Plasma (Peripheral Venous Blood): AUC_inf
  Organism|PeripheralVenousBlood|Caffeine|Plasma (Peripheral Venous Blood): AUC_inf_norm
  Organism|PeripheralVenousBlood|Caffeine|Plasma (Peripheral Venous Blood): MRT
  Organism|PeripheralVenousBlood|Caffeine|Plasma (Peripheral Venous Blood): Thalf
  Organism|PeripheralVenousBlood|Caffeine|Plasma (Peripheral Venous Blood): FractionAucLastToInf
  Organism|PeripheralVenousBlood|Caffeine|Plasma (Peripheral Venous Blood): CL
  Organism|PeripheralVenousBlood|Caffeine|Plasma (Peripheral Venous Blood): Vss
  Organism|PeripheralVenousBlood|Caffeine|Plasma (Peripheral Venous Blood): Vd